### ISSUE TITLE
[BUG] Auto generated Client side Java source files missing "import java.util.Arrays"

### ISSUE THREAD

**Issue by wwwsrikanth** [OpenAPITools/openapi-generator#8408]
#### Bug Report Checklist

- [ ] Have you provided a full/minimal spec to reproduce the issue?
- [ ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [ ] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [ ] Have you searched for related issues/PRs?
- [ ] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

Auto generated client side java source files missing "import java.util.Arrays". This is causing compilation issues while generating client side stub jars. For example the Attachment.java generated is missing "import java.util.Arrays"

##### openapi-generator version

openapi-generator-4.1.1.jar

##### OpenAPI declaration file content or url

<!-- if it is a bug, a json or yaml that produces it.
If you post the code inline, please wrap it with
```yaml
openapi: 3.0.1
info:
  title: Log Service
  description: 'This is Log Service Restful API design. '
  termsOfService: http://swagger.io/terms/
  contact:
    email: creators@xyz.com
  license:
    name: Apache 2.0
    url: http://www.apache.org/licenses/LICENSE-2.0.html
  version: 1.0.0
externalDocs:
  description: Find out more about Swagger
  url: http://swagger.io
servers:
  - url: https://localhost:8081/log/v1
  - url: http://localhost:8081/log/v1
paths:
        
  /api/notification/attachments:
    post:
      tags:
        - attachment
      summary: Upload attachment
      operationId: uploadAttachment
      parameters:
        - name: x-tenant-id
          in: header
          description: Id of tenant
          schema:
            type: string
            format: uuid
          required: true
      requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                file:
                  type: string
                  format: binary
        required: true
      responses:
        201:
          description: Successful operation
          content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Attachment'

components:
  schemas:

    Attachment:
      type: object
      properties:
        name:
          type: string
          description: "Name of the file uploaded"
        id:
          type: string
          format: uuid
          description: "Id of the attachment"
        contents:
          type: string
          format: byte
          description: "Id of the attachment"
```
##### Generation Details

Gradle task used to generate client side stubs 
task genClient {
    inputs.file(swaggerInput)
    outputs.dir(swaggerClientOutputDir)
    doLast {
        def config = new CodegenConfigurator()
        config.setInputSpec(swaggerInput)
        config.setOutputDir(swaggerClientOutputDir)
        config.setGeneratorName('java')
        config.setTemplateDir(templateDir)

        config.setAdditionalProperties([
                'invokerPackage' : invokerPackage_client,
                'modelPackage'   : modelPackage_client,
                'apiPackage'     : apiPackage_client,
                'dateLibrary'    : dateLibrary,
                "groupId"        : rootPackage,
                "artifactId"     : clientArtifactId,
                "artifactVersion": packageVersion,
                "library"        : "resttemplate",
        ])
        new DefaultGenerator().opts(config.toClientOptInput()).generate()
    }
}

##### Steps to reproduce

./gradlew clean genClient

##### Suggest a fix

While generating client side java stubs, add import statement "import java.util.Arrays", wherever applicable.

**Comment by wwwsrikanth** [comment]
[audit_service_swagger.txt](https://github.com/OpenAPITools/openapi-generator/files/5795635/audit_service_swagger.txt)

**Comment by wing328** [comment]
Can you try the latest stable version v5.0.0 instead as mentioned in the issue template?

**Comment by wwwsrikanth** [comment]
> Can you try the latest stable version v5.0.0 instead as mentioned in the issue template?

Yes the issues is there with the latest stable version v5.0.0. Just verified

**Comment by wwwsrikanth** [comment]
HI @wing328  any update on this issue ?

**Comment by wwwsrikanth** [comment]
HI @wing328 any update on this issue ?
